.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
```

Comment:
improve what? ;)

Reply:
OK, You've got me, I copied that from GH's default bug report template.